src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        for (int i = 0; i < AMOUNT_OF_MODELS; i++) {
```

Comment:
The `AMOUNT_OF_MODELS` constant is not defined within this class. Ensure that you have a constant defined, or if it's supposed to be passed into this class, make sure it's available as a class member or through a constructor parameter.